app/controllers/pg_hero/home_controller.rb

```diff
@@ -275,6 +275,15 @@ def connections
 
       @connections_by_database = group_connections(@connection_sources, :database)
       @connections_by_user = group_connections(@connection_sources, :user)
+
+      @citus_enabled = @database.citus_enabled?
+      if @citus_enabled
+        @citus_worker_connection_sources = @database.citus_worker_connection_sources
+        @citus_worker_total_connections = @database.citus_worker_total_connections
+
+        @citus_worker_connections_by_database = @citus_worker_connection_sources.map {|item| group_connections(item, :database)}
```

Comment:
Can we use better name for this (and other instances) instead of `item`? `c` for connection is OK for example. (In Ruby, using abbreviations in this places is kinda convention so it is OK to use single letter variable name.)